templates/layout/parts/profile_selector.html.twig

```diff
@@ -34,215 +34,234 @@
+                </a>
+                <a href="{{ target }}?active_entity=all" class="btn btn-secondary"
+                    title="{{ __('Select all') }}" data-bs-toggle="tooltip" data-bs-placement="top">
+                    <i class="ti ti-eye"></i>
+                </a>
+                </div>
+            </form>
+
+            <div class="fancytree-grid-container flexbox-item-grow entity_tree">
+                <table id="tree_entity{{ rand }}">
+                <colgroup>
+                    <col></col>
+                </colgroup>
+                <thead>
+                    <tr>
+                        <th class="parent-path"></th>
+                    </tr>
+                </thead>
+                <tbody>
+                </tbody>
+                </table>
+                <div id="verticalScrollbar-{{ rand }}" style="height:100%;"></div>
+            </div>
+        </div>
+    </div>
+
+    <script type="text/javascript">
+    $(function() {
+        var initTree{{ rand }} = function() {
+            if ($.ui.fancytree.getTree("#tree_entity{{ rand }}") !== null) {
+                return;
             }
-         }, 500);
-      }).focus();
-
-      $('#entsearchtext{{ rand }}_clear').click(function () {
-         $('#entsearchtext{{ rand }}').val('');
-         searchTree();
-      });
-
-      // when the shortcut for entity form is called
-      hotkeys('ctrl+alt+e, option+command+e', async function(e) {
-         e.stopPropagation();
-         e.preventDefault();
-         $('.user-menu-dropdown-toggle').dropdown('show');
-         await new Promise(r => setTimeout(r, 100));
-         $('.entity-dropdown-toggle').dropdown('show');
-         $('input[name=entsearchtext]').filter(":visible")[0].focus();
-      });
-   });
-   </script>
+
+            $('#tree_entity{{ rand }}').fancytree({
+                // load plugins
+                extensions: ['filter', 'glyph', 'grid'],
+
+                // Scroll node into visible area, when focused by keyboard
+                autoScroll: true,
+
+                // enable font-awesome icons
+                glyph: {
+                    preset: "awesome5",
+                    map: {}
+                },
+
+                // enable virtual dom, it requires the grid (table extension) plugin
+                table: {
+                    indentation: 20,       // indent 20px per node level
+                    nodeColumnIdx: 0,      // render the node title into the 1st column
+                    mergeStatusColumns: false,
+                },
+                grid: {
+                    mergeStatusColumns: false,
+                },
+                viewport: {
+                    enabled: true,
+                    count: 15, // number of items to display at once
+                },
+
+                // translate strings
+                strings: {
+                    loading: __("Loading..."),
+                    loadError: __("Load error!"),
```

Comment:
The other strings already exists in GLPI. To not have to translate a new string, we could reuse this one.
```suggestion
                    loadError: __("Unexpected error."),
```